### ISSUE TITLE
2D geometry needs 0.5 vDetectorOffset??

### ISSUE THREAD

**Issue by AnderBiguri** [CERN/TIGRE#208]
Investigate this.

**Comment by AnderBiguri** [comment]
Code from #207 : 

```
%% Init 
clear;
close all;
%% Test parameters
test_case = 1

if test_case == 0
    % Both even
    nVoxelZ=10;
    nDetectorV=10;
elseif test_case == 1
    % nVoxel Even, nDetec Odd
    nVoxelZ=10;
    nDetectorV=11;
elseif test_case == 2
    % nVoxel Odd, nDetec Even
    nVoxelZ=11;
    nDetectorV=10;
else
    % nVoxel Odd, nDetec Odd
    nVoxelZ=11;
    nDetectorV=11;
end

nVoxelX = nVoxelZ;
nVoxelY = nVoxelZ;
nDetectorU = nDetectorV;

nangles=10;

mode='cone';  % 'parallel'
ptype= 'Siddon'; % 'interpolated' % 'ray-voxel'
%%
geo=defaultGeometry('mode',mode);
geo.DSD = 2000;
geo.DSO = 1000;
geo.nVoxel = [nVoxelX;nVoxelY;nVoxelZ];
geo.dVoxel = [1.0; 1.0; 1.0];
geo.sVoxel = geo.nVoxel .* geo.dVoxel;
geo.dDetector = [2;2] ;              % size of each pixel            (mm)
geo.nDetector = [nDetectorU; nDetectorV];     % has to be true 3D for python version
geo.sDetector = geo.dDetector .* geo.nDetector;
geo.offOrigin = [0;0;0];
geo.offDetector = [0.0; 0.0] ;
geo.rotDetector = [0; 0; 0];
geo.accuracy=0.01;

%%
angles = linspace(0, 2 * pi - 2*pi/nangles, nangles);

angles=[0,pi/2,pi,3/2*pi]
wire = zeros(geo.nVoxel', 'single');
wire(int32((geo.nVoxel(1))/2):int32((geo.nVoxel(1)+1)/2), int32((geo.nVoxel(2))/2):int32((geo.nVoxel(2)+1)/2), :) = 1;
%%
proj = Ax(wire,geo,angles, ptype);

plotProj(proj, angles,'slice',1)
```

Shows that for  test_case == 1 the projections are not symmetric. This is likely related to why #207 does not work well. 
I have not been able to reproduce for another test case, this is likely related to some bug at `compute_deltas` functions.

**Comment by AnderBiguri** [comment]
The above is caused by the solution to #198. Possibly #218 has actually fixed #198 in a less hacky way and we can remove the artificial offsets.
